app/requests_handlers.py

```diff
@@ -1261,16 +1261,15 @@ async def get(self, request):
         self.setup_request(request)
 
         display_region = 'ni'
-        page_title = 'You need to visit the Communal Establishment Portal'
+        page_title = 'You need to visit the Communal Establishment Manager Portal'
         locale = 'en'
 
         self.log_entry(request, display_region + '/requests/access-code/ce-manager')
 
         return {
                 'page_title': page_title,
                 'display_region': display_region,
```

Comment:
display_region not used in template and could be removed.

Reply:
I'll remove it from line 1271 but leave it in line 1263 as it's being used in the log_entry (and I guess it might get used in the template possibly in future). Is that ok, or should I replace the log entry with 'ni/requests/access-code/ce-manager' hard coded?